src/chainstate/burn/operations/peg_in.rs

```diff
@@ -76,11 +78,17 @@ impl PegInOp {
 
             0      2  3                  24                            64       80
```

Comment:
I guess we need an extra byte for the size prefix
```suggestion
            0      2  3                  24                            65       80
```

Reply:
Just to confirm, we want to take a byte from the memo and not make the data 81 bytes long?

Reply:
Yes. We only have 80 bytes. The memo is not important and is just a field to absorb any remaining bytes unused by the protocol.

Reply:
Confirmed.